Tasks/PyPIPublisherV0/task.json

```diff
@@ -1,10 +1,10 @@
 {
     "id": "2d8a1d60-8ccd-11e7-a792-11ac56e9f553",
     "name": "PyPIPublisher",
-    "friendlyName": "PyPI Publisher",
-    "description": "Create and upload an sdist or wheel to a PyPI-compatible index using Twine.",
+    "friendlyName": "PyPI publisher",
+    "description": "Create and upload an sdist or wheel to a PyPI-compatible index using twine",
```

Comment:
Shouldn't Twine be capitalized?

Reply:
I changed it because its [product homepage](https://pypi.org/project/twine/) consistently refers to the tool in its lowercase form.  Please shout if you disagree.

Reply:
![image](https://user-images.githubusercontent.com/26548919/56372224-999df780-61cc-11e9-96b7-b12691a7e70f.png)
Seems inconsistent w/in the page.  I don't have strong feelings on the matter.  If I were to pick one over the other, I personally would capitalize it.